server/controllers/devices.coffee

```diff
@@ -122,32 +128,43 @@ createDevice = (device, cb) ->
 
 # Update device :
 #       * update device access
-updateDevice = (oldDevice, device, cb) ->
+updateDevice = (oldDevice, device, callback) ->
+
     path = "request/access/byApp/"
-    clientDS.post path, key: oldDevice.id, (err, result, accesses) ->
+    clientDS.post path, key: oldDevice._id, (err, result, accesses) ->
```

Comment:
Fix error when you update permission device.
Actually it create a new device with new password... now it's updated